pkg/controllers/cloud_config_sync_controller.go

```diff
@@ -308,3 +338,46 @@ func (r *CloudConfigReconciler) setDegradedCondition(ctx context.Context) error
 	klog.Info("Cloud Config Controller is degraded")
 	return r.syncStatus(ctx, co, conds)
 }
+
+func (r *CloudConfigReconciler) setUpgradeableCondition(ctx context.Context, condition configv1.ConditionStatus, reason string, message string) error {
+	co, err := r.getOrCreateClusterOperator(ctx)
+	if err != nil {
+		return err
+	}
+
+	conds := []configv1.ClusterOperatorStatusCondition{
+		newClusterOperatorStatusCondition(configv1.OperatorUpgradeable, configv1.ConditionFalse, reason, message),
```

Comment:
I think it should put the input argument "condition" in the conds, instead of the hard-coded configv1.ConditionFalse. This code line should be: 

newClusterOperatorStatusCondition(configv1.OperatorUpgradeable, condition, reason, message),

Reply:
yes, good spot @yanhua121 . i will update